scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh

```diff
@@ -51,6 +51,7 @@ RAWLS_SA=$(docker run -e VAULT_TOKEN="$(cat ~/.vault-token)" -it broadinstitute/
 CROMWELL_SA=$(docker run -e VAULT_TOKEN="$(cat ~/.vault-token)" -it broadinstitute/dsde-toolbox:dev vault read -field="client_email" secret/dsde/firecloud/${vault_env}/cromwell/cromwell-account.json)
 gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=serviceAccount:"${RAWLS_SA}" --role=roles/editor
 gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=serviceAccount:"${RAWLS_SA}" --role=roles/resourcemanager.projectMover
+gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=serviceAccount:"${RAWLS_SA}" --role=roles/resourcemanager.projectIamAdmin # required to set policies on google projects from the Buffer service
```

Comment:
Big upvote on comments, especially for stuff like this.  There are permissions we grant that are obvious, but that's not always the case.  Especially if there's specific permissions that we are _not_ granting for a specific reason, that's probably even more important, "It might be tempting to grant X, but if we did that, then users would be able to do ABC and easily break their Workspaces" or something like that.

I'm not saying to go around and add those comments now to all the other grants, but something to keep in mind.

Reply:
agreed!